File: fixtures/scale/swift/large/Families/LcgStream/LcgStreamRunner.swift
// Correctness entry point for the lcgstream family; wraps CorrectnessEngine
// with this family's cipher and known-answer vectors.
struct LcgStreamRunner {
    private let engine = CorrectnessEngine()

    func check() -> VectorOutcome {
        let cipher = LcgStreamCipher()
        let vectors = LcgStreamVectors.all()
        return engine.verify(cipher, vectors)
    }
}

File: fixtures/scale/php/large/src/Families/LcgStream/LcgStreamRunner.php
<?php

namespace CipherLab\Families\LcgStream;

use CipherLab\Abstractions\VectorOutcome;
use CipherLab\Engine\CorrectnessEngine;

// Correctness entry point for the lcgstream family; wraps CorrectnessEngine
// with this family's cipher and known-answer vectors.
class LcgStreamRunner
{
    private CorrectnessEngine $engine;

    public function __construct()
    {
        $this->engine = new CorrectnessEngine();
    }

    public function check(): VectorOutcome
    {
        return $this->engine->verify(new LcgStreamCipher(), LcgStreamVectors::all());
    }
}
